### ISSUE TITLE
Update to focalboard 7.10.4 loses attachents

### ISSUE THREAD

**Issue by AuspeXeu** [mattermost/mattermost#23792]
![image](https://github.com/mattermost/mattermost/assets/2115696/39113c7f-8cd9-4f60-8964-a4d6beb238b5)

![image](https://github.com/mattermost/mattermost/assets/2115696/387c4fba-cb47-48cb-b067-f673f855324e)

![image](https://github.com/mattermost/mattermost/assets/2115696/0cf447c0-924d-4d5d-8dd8-25833043e5f1)

**Comment by AuspeXeu** [comment]
Does this help?
![image](https://github.com/mattermost/mattermost/assets/2115696/370b49a6-23a5-42ff-abd7-8311a17ba07d)

The observed behaviour is that the attachments are missing as per the screenshots above.
I cannot reproduce it since the newly created cards are fine, so it seems to me that a migration was/ist broken.

**Comment by amyblais** [comment]
Thanks, opened https://mattermost.atlassian.net/browse/MM-53240.

**Comment by AuspeXeu** [comment]
This is now also happening with newly created attachments!
```
{"timestamp":"2023-07-03 06:39:00.534 Z","level":"error","msg":"file not found","caller":"app/plugin_api.go:976","plugin_id":"focalboard"}
{"timestamp":"2023-07-03 06:39:00.536 Z","level":"error","msg":"API ERROR","caller":"app/plugin_api.go:976","plugin_id":"focalboard","code":"500","error":"\"file not found\"","api":"/api/v2/files/teams/r63a14yqwfnj5mkw1r6xw6iudh/b56nk46no3f899gayd9pryw37bw/7qmfs9uohatrepfh4rsoojw979o.png"}
```

**Comment by amyblais** [comment]
Notes from a dev in the Jira ticket:

The fix was backported to v7.10. We won’t cut a dot release for it, but if we have to cut another one in the future it will be included. The fix is in boards v7.11, which will release with mattermost v8.0.

**Comment by maglevize** [comment]
I have the same issue, after docker update images to 7.10.4-rc, all old image on boards can't be downloaded.
I try to downgrade to 7.10.3 but to no fixed.

then I find this issue, follow the update from Jira Ticket, said solved on v8.0-rc2. then I straight updated to 8.0.0-rc2
<img width="590" alt="image" src="https://github.com/mattermost/mattermost/assets/71497388/4a804b4a-0d16-4b6f-aa88-71fa15c45b3a">

<img width="784" alt="image" src="https://github.com/mattermost/mattermost/assets/71497388/d677e3fb-972d-4dfe-9b4e-488af6ade441">
issues still there.

<img width="776" alt="image" src="https://github.com/mattermost/mattermost/assets/71497388/8e8a5d07-5e4e-4d5e-94ef-a0f4f30f8f2d">
Newly uploaded image (After migration) is just fine
